plugin/src/main/groovy/com/bmuschko/gradle/tomcat/tasks/AbstractTomcatRun.groovy

```diff
@@ -106,6 +107,15 @@ abstract class AbstractTomcatRun extends Tomcat {
     @InputFiles
     Iterable<File> additionalRuntimeResources = []
 
+    /**
+     * Defines additional web file or directories that are not provided by the web application.
+     * In task TomcatRunWar, you can add src/main/webapp as liveedit web resources.
+     *
+     * exclue: WEB-INF/lib, WEB-INF/classes, WEB-INF/web.xml, META-INF/
+     */
+    @InputFiles
+    Iterable<File> additionalWebResources = []
```

Comment:
Doesn't this already work if you add the files to `additionalRuntimeResources`?

Reply:
`additionalRuntimeResources` add resource files to classpath, Right?

`additionalWebResources` add static files to webapp context root ( / ), not WEB-INF/classes.